lib/sqlalchemy/engine/reflection.py

```diff
@@ -469,6 +469,11 @@ def get_indexes(self, table_name, schema=None, **kw):
         unique
           boolean
 
+        column_sorting
+          optional dict mapping column names to dict of sorting options.
+          each sorting options dict may set any of the following keywords
+          to True: ``asc``, ``desc``, ``nullsfirst``, ``nullslast``.
+
```

Comment:
add a version note, this should be good for 1.3 so let's assume 1.3.5:

`    .. versionadded:: 1.3.5`